Makefile

```diff
@@ -0,0 +1,88 @@
+XDR_BASE_URL_CURR=https://github.com/stellar/stellar-core/raw/master/src/protocol-curr/xdr
+XDR_BASE_LOCAL_CURR=xdr/curr
+XDR_FILES_CURR= \
+	Stellar-SCP.x \
+	Stellar-ledger-entries.x \
+	Stellar-ledger.x \
+	Stellar-overlay.x \
+	Stellar-transaction.x \
+	Stellar-types.x
+XDR_FILES_LOCAL_CURR=$(addprefix xdr/curr/,$(XDR_FILES_CURR))
```

Comment:
This whole `curr` and `next` thing broke the entire ecosystem, I bet. Why couldn't they keep `xdr/` and add `xdr-next/`??